src/bluetooth/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod detector;
```

Comment:
@karlpet can you please enable adding newlines in your editor?
I don't think it will be an issue for us but GitHub complains :)